build.gradle.kts

```diff
@@ -100,7 +100,7 @@ val generateHtmlReportJavaScript = tasks.register<NpxTask>("generateHtmlReportJa
     val output = outputDir.map { it.file("org/gradle/profiler/report/report.js") }
     inputs.file(source)
     outputs.dir(outputDir)
-    command = "browserify"
+    command = "browserify@16.5.2"
```

Comment:
Why is this better? Isn't the lock-file better so we also lock in all the transitive dependencies?

Reply:
I think this is fine like this, and it's fewer changes to the codebase. I don't expect this to break in a way that would otherwise be prevented by putting everything in a lock file. If it does, we can add back the lock file.

Reply:
Fine with me.